COMPATIBILITY.md

```diff
@@ -35,6 +35,24 @@ To see what that means in practice, you can use https://browserslist.dev
 <a href="https://saucelabs.com/u/sinonjs"><img src="https://saucelabs.com/browser-matrix/sinonjs.svg" alt="Sauce Test Status"></a>
 </p>
 
+## Previous supported Node.js
+
+The following table serves as an orientation if support for an older unsupported Node.js version is required.
+
+| Sinon.JS | supported Node.js |
+|:--------:|:-----------------:|
+|    18    |    18, 20, 22     |
```

Comment:
```suggestion
|    18    |    18, 20     |
```
You can use  https://browserslist.dev/?q=bWFpbnRhaW5lZCBub2RlIHZlcnNpb25z or use `npx browserslist` directly to see that 22 is not yet in maintenance phase. We still run version 22 on CI to check that it works, but it's not _supported_ yet :)

Reply:
Actually, I just checked, and that web page is not consistent with the `npx browserslist` output, which shows 3 versions.